Packs/KnowBe4KMSAT/Integrations/KnowBe4KMSAT/KnowBe4KMSAT.py

```diff
@@ -0,0 +1,1348 @@
+Linting: https://xsoar.pan.dev/docs/integrations/linting
+
+This is an empty structure file. Check an example at;
+https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py
+
+"""
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+import traceback
+import urllib3
+from typing import Dict
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+""" CONSTANTS """
+
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"  # ISO8601 format with UTC, default in XSOAR
+
+""" CLIENT CLASS """
+
+
+class Client(BaseClient):
+    """KMSAT Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def __init__(self, base_url, verify, proxy, headers=None):
+        headers["X-KB4-Integration"] = "Cortex XSOAR KMSAT"
+        super().__init__(base_url=base_url, verify=verify, headers=headers, proxy=proxy)
+
+    def kmsat_account_info(self):
+        """ Returns account info
+
+        Returns:
+            dict: HTTP Response
+        """
+        return self._http_request(
+            method="GET", url_suffix="/account", resp_type="json", ok_codes=(200,)
+        )
+
+    def kmsat_account_risk_score_history(self, params: dict):
+        """ Returns account risk score history
+
+        Args:
+            params (dict): Params for account risk score history
+
+        Returns:
+            dict: HTTP Response
+        """
+        return self._http_request(
+            method="GET",
+            url_suffix="/account/risk_score_history",
+            resp_type="json",
+            ok_codes=(200,),
+            params=params,
+        )
+
+    def kmsat_groups_list(self, params: dict):
+        """ Returns groups
+
+        Args:
+            params (dict): Params for groups risk score history
+
+        Returns:
+            dict: HTTP Response
+        """
+        return self._http_request(
+            method="GET",
+            url_suffix="/groups",
+            resp_type="json",
+            ok_codes=(200,),
+            params=params,
+        )
+
+    def kmsat_groups_risk_score_history(self, group_id: int, params: dict):
+        """ Returns groups risk score history
+
+        Args:
+            group_id (int): Group ID
+            params (dict): Params for groups risk score history
+
+        Returns:
+            dict: HTTP Response
+        """
+        return self._http_request(
+            method="GET",
+            url_suffix=f"/groups/{group_id}/risk_score_history",
+            resp_type="json",
+            ok_codes=(200, 404,),
```

Comment:
Regarding the 404, Don't you prefer to let the user know that there is no ID (group id in this case) rather than return an empty response? The user might think that the ID is correct but the results are empty (risk score history in this case)?

Reply:
I removed the 404s to let original error handle appropriately.